linux_os/guide/services/sssd/sssd_offline_cred_expiration/oval/shared.xml

```diff
@@ -4,7 +4,7 @@
     <criteria operator="OR">
       <criterion comment="Check offline_credentials_expiration in /etc/sssd/sssd.conf"
       test_ref="test_sssd_offline_cred_expiration" />
-      {{% if product in ["ol8", "rhel8"] %}}
+      {{% if product in ["ol8", "ol9", "rhel8", "rhel9"] %}}
```

Comment:
Could someone of @ComplianceAsCode/red-hatters validate this is also applicable to rhel9

It seems like it should be from here: https://www.stigviewer.com/stig/red_hat_enterprise_linux_9/2023-12-01/finding/V-258133

Reply:
I can confirm. Thanks